json/ru.json

```diff
@@ -25,12 +31,15 @@
   "Send Password Reset Link": "Отправить ссылку сброса пароля",
   "Service Unavailable": "Сервис недоступен",
   "Sorry, the page you are looking for could not be found.": "Извините, страница, которую вы ищете, не найдена.",
+  "Sorry, you are forbidden from accessing this page.": "Извините, вам запрещено пользоваться этой страницей.",
   "Sorry, you are making too many requests to our servers.": "Извините, вы делаете слишком много запросов на наши серверы.",
   "Sorry, you are not authorized to access this page.": "К сожалению, у вас нет доступа к этой странице.",
   "Sorry, your session has expired. Please refresh and try again.": "К сожалению, срок действия вашей сессии истек. Обновите и повторите попытку.",
   "Sorry, we are doing some maintenance. Please check back soon.": "Извините, мы проводим некоторые работы. Пожалуйста, зайдите позже.",
+  "Toggle navigation": "Переключить навигацию",
   "Too Many Requests": "Слишком много запросов",
   "Unauthorized": "Не авторизован",
+  "Verify Email Address": "Проверьте email-адрес",
```

Comment:
![image](https://user-images.githubusercontent.com/13885653/48301368-0f63b000-e517-11e8-9e4a-7dc7229878d5.png)
Имеется в виду действие. Будет более правильно "Верифицировать email-адрес".

Reply:
Какой конфуз. Отправил пул с исправлениями (#941).

Reply:
Еще конфуз! Недавно заметил что заголовок письма использует ту же строку. На заголовок бы более подоходил призыв к действию: `Подтвердите email-адрес`. Получается строка одна, а на русском ее нужно раздваивать. Тут только новую строку создавать и отправлять PR laravel/framework. @andrey-helldar, что думаете?

Reply:
Логика да, для заголовка предпочтительней "Подтвердите email-адрес", а для кнопки - "Подтвердить email-адрес".
В пул-реквесте caouecs/Laravel-lang#941 я исправил текст на:
`"Verify Email Address": "Подтвердить email-адрес",`
Для заголовка, конечно, это не самый лучший вариант, но писать "Проверьте" на кнопке - бред.

Что касается разделения полей, Тейлор вряд ли согласится на это. Он и более серьезные вещи по бороде пускает. Сталкивался.
В случае с письмом, на мой взгляд, лучшим вариантом будет создание своего файла переводов и добавлением в него нужных ключей.
Именно по этой причине написал пакет [andrey-helldar/lang-translations](https://github.com/andrey-helldar/lang-translations), чтобы копипастить файлы из проекта в проект.

Reply:
Не факт, но уверен что Тейлор может отклонить такой пул-реквест еще и по той причине, что ему, мягко сказать, по-барабану на русский язык. Мол, "в английском одна фраза применяется и не выпендривайтесь".

Reply:
Тоже сталкивался, поэтому пытаться наверное не буду. Спасибо за пакет.

Reply:
По поводу пакета - в отличие от Тейлора, добавляю в него действительно необходимые вещи.
Например, надо разделить 2 записи т.к. в русском языке они отличаются - вуаля :)
Так что, если что - реквесты принимаются.

Reply:
Похоже, я нашел решение проблемы читая [доку](https://laravel.com/docs/5.7/localization#overriding-package-language-files).
Напомню суть разговора: в английском языке строка `Verify Email Address` подходит как для заголовка, так и для кнопки, а в русском - нет.

Цитата из доки:
> Some packages may ship with their own language files. Instead of changing the package's core files to tweak these lines, you may override them by placing files in the  `resources/lang/vendor/{package}/{locale}` directory.

Перевод:
> Некоторые пакеты могут использовать свои файлы локализаций. Вместо изменения файлов пакета, Вы можете легко их переопределить, разместив в папку `resources/lang/vendor/{package}/{locale}`.

Reply:
Круто!
А я и не заметил. Спасибо.